x-pack/plugins/discover_enhanced/ui_tests/tests/value_suggestions.spec.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout';
+import { test } from '../fixtures';
+
+test.describe('Discover app - value suggestions', () => {
+  test.beforeAll(async ({ esArchiver, kbnClient }) => {
+    await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+    await kbnClient.importExport.load(
+      'x-pack/test/functional/fixtures/kbn_archiver/dashboard_drilldowns/drilldowns'
+    );
```

Comment:
It would be nice at some point to enumerate these options better, at least the most common ones.

```ts
await esArchiver.loadIfNeeded('logstash_functional');
// or
await esArchiver.load.logstashFunctional();
```